u7-components/src/router/index.ts

```diff
@@ -0,0 +1,39 @@
+import Vue from 'vue'
+import VueRouter, { RouteConfig } from 'vue-router'
+import Home from '@/views/Home.vue'
+import VideoPlayer from '@/views/VideoPlayer.vue'
+import BooksSearch from '@/views/BookSearch.vue'
+import BookDetails from '@/views/BookDetails.vue'
+
+Vue.use(VueRouter)
+
+const routes: Array<RouteConfig> = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/videos/:videoId',
+    name: 'VideoPlayer',
+    component: VideoPlayer
+  },
+  {
+    path: '/books',
+    name: 'BookSearch',
+    component: BooksSearch
+  },
+  {
+    path: '/books/:bookId',
+    name: 'BookDetails',
+    component: BookDetails
+  }
```

Comment:
ดี. โปรดใช้ความระมัดระวังในการเรียกพารามิเตอร์ `bookId` (เพราะมันเรียกว่า `isbn13` บนวัตถุ book) พวกเขาอาจแตกต่างกัน แต่มันอาจจะสับสน
Good. Be careful calling the parameter `bookId` (because it's called `isbn13` on the book object). They can be different; but it might be confusing.